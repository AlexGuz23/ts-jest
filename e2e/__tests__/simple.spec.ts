import configureTestCase from '../__helpers__/test-case';

describe('Simple test', () => {
  it('should pass with babel 6', () => {
    const testCase = configureTestCase('simple');
    expect(testCase.run(0).status).toBe(0);
  });
  it('should pass with babel 7', () => {
    const testCase = configureTestCase('simple', { template: 'with-babel-7' });
    expect(testCase.run(0).status).toBe(0);
  });
  it('should pass with jest 22', () => {
    const testCase = configureTestCase('simple', { template: 'with-jest-22' });
    expect(testCase.run(0).status).toBe(0);
  });
});
