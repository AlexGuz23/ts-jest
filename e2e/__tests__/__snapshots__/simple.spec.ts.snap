// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple test should pass with each template 1`] = `
Object {
  "default": jest exit code: 0
    ===[ STDOUT ]===================================================================
    
    ===[ STDERR ]===================================================================
    PASS __tests__/Hello.test.ts
      Hello Class
        P should create a new Hello
    
    Test Suites: 1 passed, 1 total
    Tests:       1 passed, 1 total
    Snapshots:   0 total
    Time:        XXs
    Ran all test suites.
    ================================================================================,
  "with-babel-7": jest exit code: 0
    ===[ STDOUT ]===================================================================
    
    ===[ STDERR ]===================================================================
    PASS __tests__/Hello.test.ts
      Hello Class
        P should create a new Hello
    
    Test Suites: 1 passed, 1 total
    Tests:       1 passed, 1 total
    Snapshots:   0 total
    Time:        XXs
    Ran all test suites.
    ================================================================================,
  "with-jest-22": jest exit code: 0
    ===[ STDOUT ]===================================================================
    
    ===[ STDERR ]===================================================================
    PASS __tests__/Hello.test.ts
      Hello Class
        P should create a new Hello
    
    Test Suites: 1 passed, 1 total
    Tests:       1 passed, 1 total
    Snapshots:   0 total
    Time:        XXs
    Ran all test suites.
    ================================================================================,
}
`;
